code/datums/components/butchering.dm

```diff
@@ -114,15 +114,15 @@
 		var/obj/sinew = V
 		var/amount = meat.guaranteed_butcher_results[sinew]
 		for(var/i in 1 to amount)
-			results += new sinew (T)
+			results +=  new sinew (T)
 		meat.guaranteed_butcher_results.Remove(sinew)
 
-	for(var/obj/item/carrion in results)
+	for(var/obj/item/food/carrion in results)
```

Comment:
This'll break things that get passed in which have the meat mat, but aren't meat.
Do an `if(istype(carrion, /obj/item/food))` check instead

Reply:
I will webedit this I swear to god

Reply:
TLALLLLLLLLL